mycore.org/config/_default/menus.de.yaml

```diff
@@ -39,11 +39,12 @@ main:
 -       {identifier: mir_install,             parent: apps_mir,           name: Installieren,                  weight: -99,      url: /documentation/apps/mir/mir_install}
 -       {identifier: mir_customize,           parent: apps_mir,           name: Individualisieren,             weight: -98,      url: /documentation/apps/mir/mir_customize}
 -       {identifier: mir_migration,           parent: apps_mir,           name: Migrieren,                     weight: -97,      url: /documentation/apps/mir/migration}
--         {identifier: migrate_mir2023_06,      parent: mir_migration,      name: MIR 2022.06 ⇒ 2023.06,       weight: -96,      url: /documentation/apps/mir/migration/migrate_mir2023_06}
--         {identifier: migrate_mir2022_06,      parent: mir_migration,      name: MIR 2021.06 ⇒ 2022.06,       weight: -95,      url: /documentation/apps/mir/migration/migrate_mir2022_06}
--         {identifier: migrate_mir2021_06,      parent: mir_migration,      name: MIR 2020.06 ⇒ 2021.06,       weight: -94,      url: /documentation/apps/mir/migration/migrate_mir2021_06}
+-         {identifier: migrate_mir2024_06,      parent: mir_migration,      name: MIR 2023.06 ⇒ 2024.06,       weight: -96,      url: /documentation/apps/mir/migration/migrate_mir2024_06}
+-         {identifier: migrate_mir2023_06,      parent: mir_migration,      name: MIR 2022.06 ⇒ 2023.06,       weight: -95,      url: /documentation/apps/mir/migration/migrate_mir2023_06}
+-         {identifier: migrate_mir2022_06,      parent: mir_migration,      name: MIR 2021.06 ⇒ 2022.06,       weight: -94,      url: /documentation/apps/mir/migration/migrate_mir2022_06}
+-         {identifier: migrate_mir2021_06,      parent: mir_migration,      name: MIR 2020.06 ⇒ 2021.06,       weight: -93,      url: /documentation/apps/mir/migration/migrate_mir2021_06}
 -         {identifier: migrate_mir2020_06,      parent: mir_migration,      name: MIR 2019.06 ⇒ 2020.06,       weight: -93,      url: /documentation/apps/mir/migration/migrate_mir2020_06}
```

Comment:
two pages now have weight `-93`, not sure if the ordering works correctly with that. `migrate_mir2020_06` could simply have weight `-92`, since there appears to be a gap anyways.

Reply:
The identical weight does not seem to have a bad impact but I fixed it anyway.